### DISCUSSION TITLE
variable for linux user

### DISCUSSION THREAD

**Discussion by crydotsnake** [i3/i3#5629] [Q&A]
Hello!

In my i3 configuration i have multiple places with a path that contains my username. Since my username can be different from time to time, i thought about creating a variable like so:

```
set $username $USER
```

Because `$USER` contains the current linux username.

and then in my paths:

```
/home/$username/path/to/my/path/
```

does this work with i3 or should i solve this differently?

**Comment by waterkip** [comment]
You have $HOME for /home/$username
$USER is also known to i3:

```
bindsym $mod+e exec env > foo.bar
bindsym $mod+Shift+e exec echo $USER >> foo.bar
```

This shows what is available to i3 and your username as the last line in the foo.bar file.

**Reply by crydotsnake** [reply]
Thanks! Why didn't I think of that on my own 🤦🏽
